packages/shared-integration/src/transformers.ts

```diff
@@ -19,7 +19,8 @@ export async function applyTransformers(
     return
 
   const skipMap = new Map<string, string>()
-  let code = original
+  // There may be multiple @apply statements in one line. In this case, magic-string will process the calculation and modify the same position. Add a space here to handle this.
+  let code = original.replace(/;(@apply|--uno|--at-apply) /g, '; $1 ')
```

Comment:
That would shift the position, which defeats the point of having source maps. Also, those values are customizable. I'd say it would be better to fix the position issue of magic-string usage.